docs/openapi.yaml

```diff
@@ -774,17 +774,32 @@ paths:
           type: boolean
           default: false
 
-  /extended/v1/contract/{contract_id}/events:
+  /extended/v1/contract/trait/contracts:
```

Comment:
To be consistent with other paths on the API (get block by height, etc.), I think this path should be: `/extended/v1/contract/by_trait`

Reply:
This endpoint will be conflicting with `/extended/v1/contract/by_trait/:contract_id`

Reply:
As long as you declare it above the variable route you'll be OK: https://stackoverflow.com/questions/31493190/express-parameterized-route-conflict

Reply:
Done!